core/frontend/src/tile/TileAdmin.ts

```diff
@@ -1170,6 +1186,24 @@ export namespace TileAdmin { // eslint-disable-line no-redeclare
      */
     disableMagnification?: boolean;
 
+    /** Allow preloading for context (reality and map tiles).
+     * This will be ignored and treated as false for mobile devices.
+     *
+     * Default value: true
+     * @alpha
+     */
+    allowPreloading?: boolean;
```

Comment:
This separate flag seems redundant - preloading could be disabled by setting the memory threshold to 0% (if you slightly adjust your `isPreloadingAllowed` implementation to compute % mem usage as 0 if no limit defined).

Reply:
Removed `isPreloadingAllowed` flag